spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -499,10 +499,20 @@ function chanUpgradeConfirm(
 
 ### Cancel Upgrade Process
 
-During the upgrade handshake a chain may cancel the upgrade by writing an error receipt into the error path and restoring the original channel to `OPEN`. The counterparty must then restore its channel to `OPEN` as well. A relayer can facilitate this by sending `ChanUpgradeCancelMsg` to the handler:
+**Sub-Definitions**:
+
+`Aborting Chain`: This is the chain that first encounters an error, writes data to the error receipt and restores the channel.
+
+**Protocol**:
+
+During the upgrade handshake an aborting chain may cancel the upgrade by writing an error receipt into the error path and restoring the original channel to `OPEN`. The counterparty must then restore its original channel to `OPEN` as well.
+
+A channelEnd may only cancel the upgrade during the upgrade negotiation process (`TRY`, `ACK`). Thus, the counterparty channel state must be either `UPGRADE_INIT` or `UPGRADE_TRY`. An upgrade cannot be cancelled on one end once the other chain has already completed its upgrade and moved to `OPEN` since that will lead the channel to being in an invalid state.
```

Comment:
```suggestion
The aborting chain may cancel the upgrade only during the upgrade negotiation process (`TRY`, `ACK`). Thus, the channel state on the counterparty chain must be either `UPGRADE_INIT` or `UPGRADE_TRY`. The aborting chain cannot cancel the channel upgrade once the counterparty chain has already completed its upgrade and moved to `OPEN`, since that will lead to the channel being in an invalid state.
```